src/app/(app)/missing-shoes/actions/get-all-missing-shoes.ts

```diff
@@ -0,0 +1,28 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+
+export const getAllMissingShoes = async () => {
+  const supabase = await getSupabaseClient();
+
+  const { data: missingShoesData, error } = await supabase
+    .from(Tables.MissingShoes)
+    .select("*");
+
+  if (error) {
```

Comment:
Why'd we return empty array on error? Shouldn't we handle error?

Reply:
I have fixed this error on client side by showing error toast